integration/fuse/src/main/java/alluxio/fuse/FuseUmountable.java

```diff
@@ -11,12 +11,16 @@
 
 package alluxio.fuse;
 
+import alluxio.jnifuse.FuseException;
+
 /**
  * An interface for unmounting the mounted Fuse applications.
  */
 public interface FuseUmountable {
   /**
    * Unmounts the Fuse application.
+   * @param force when the umount times out due to the device is busy,
+   *              it will throw exception (catched by {@link FuseSignalHandler})
```

Comment:
```
   * @param force true to throw an exception and treat umount as a no-op 
   *            when umount timeouts due to the fuse device is busy.
```
modify the previous line to fit the indentation codestyle

Reply:
done